src/static/css/page.css

```
@@ -612,6 +612,14 @@ td.numeric {
   text-align: right;
 }
 
+td .numeric-good {
+  color: #38761d;
+}
+
+td .numeric-bad {
+  color: #e00;
```

Can we make this a darker red to pass Lighthouse Accessibility tests?

```suggestion
  color: ##B60000;
```